pkg/scheduler/core/generic_scheduler.go

```diff
@@ -903,11 +904,34 @@ func pickOneNodeForPreemption(nodesToVictims map[*v1.Node]*schedulerapi.Victims)
 			lenNodes2++
 		}
 	}
+	if lenNodes2 == 1 {
+		return minNodes2[0]
+	}
+
+	// There are a few nodes with same number of pods.
+	// Find one with the latest start time, i.e. latest of earliest of all pods.
+	theLastestEarliest := metav1.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
I don't know if there is such a way of using it elsewhere, why is it not the default value of 0? It looks a bit hacking